crawler/attribution/entrypoints/prototype-overload.js

```diff
@@ -0,0 +1,21 @@
+{
+    const descriptor = Object.getOwnPropertyDescriptor(
+        Document.prototype,
+        'createElement'
+    );
+    const origImpl = descriptor.value;
+    const src = document.currentScript.src;
+    descriptor.value = function () {
+        console.log(src, navigator.userAgent);
+        window.addResult('Navigator.prototype.userAgent', 'createElement prototype overload', src);
+        return origImpl.apply(this, arguments);
+    };
+
+    Object.defineProperty(
```

Comment:
doesn't this influence other tests that use createElement?

Reply:
We could potentially run this test in an iframe to make sure it doesn't have side-effects, but not sure if worth it. Up to you.

Reply:
I thought about it, but right now running things in an iframe makes it more flaky